### PR TITLE
Updated Clockify

### PR DESCRIPTION
**Pull request by fbernhart**
clockify

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

**Comment by service-paradis** [comment]
Thanks for your work @fbernhart 😄 
The icon looks different when compared to the original found at https://clockify.me/brand-assets
image

Can you check it out?

**Comment by fbernhart** [comment]
@service-paradis I've rebuilt the icon with the official SVG file from the website. Can you please check again, if it's o.k. now? Thanks

**Comment by service-paradis** [comment]
> @service-paradis I've rebuilt the icon with the official SVG file from the website. Can you please check again, if it's o.k. now? Thanks

Looking good, thanks 💯 
Can you update the source in `simple-icons.json`?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1223,7 +1223,7 @@
         {
             "title": "Clockify",
             "hex": "03A9F4",
-            "source": "https://clockify.me/"
+            "source": "https://clockify.me/brand-assets"
```
Comment (service-paradis) [inline _data/simple-icons.json:1226]: 💯

**Pull request merged**